frontend/shared-ui/assets/styles/main.scss

```
@@ -257,6 +257,14 @@
   }
 }
 
+// Sender colours are only legible against white, so incoming email HTML is not themed.
+.dark .email-light-canvas {
+  @apply p-2 rounded border border-border;
+  color-scheme: light;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)